specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/StorageAccountCreateOrUpdateUserAssignedEncryptionIdentityWithCMK.json

```diff
@@ -0,0 +1,122 @@
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "identity": {
+          "userAssignedIdentities": {
+            "/subscriptions/{subscription-id}/resourceGroups/res9101/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{managed-identity-name}": {
+              "principalId": "8d823284-1060-42a5-9ec4-ed3d831e24d7",
+              "tenantId": "f14da12a-1e85-4c31-ae98-bcf5608d2266",
+              "clientId": "fbaa6278-1ecc-415c-819f-6e2058d3acb5"
+            }
+          },
+          "type": "UserAssigned"
+        },
+        "sku": {
+          "name": "Standard_LRS",
+          "tier": "Standard"
+        },
+        "kind": "StorageV2",
+        "id": "/subscriptions/{subscription-id}/resourceGroups/res9101/providers/Microsoft.Storage/storageAccounts/sto4445",
+        "name": "sto4445",
+        "type": "Microsoft.Storage/storageAccounts",
+        "location": "eastus",
+        "tags": {},
+        "properties": {
+          "privateEndpointConnections": [],
+          "networkAcls": {
+            "bypass": "AzureServices",
+            "virtualNetworkRules": [],
+            "ipRules": [],
+            "defaultAction": "Allow"
+        },
+        "supportsHttpsTrafficOnly": true,
+        "encryption": {
+          "identity": {
+            "type": "ManagedServiceIdentity",
+            "userAssignedIdentity": "/subscriptions/{subscription-id}/resourcegroups/res9101/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{managed-identity-name}"
+          },
+        "keyvaultproperties": {
+           "keyvaulturi": "https://myvault8569.vault.azure.net",
+           "keyname": "wrappingKey",
+           "keyversion": "",
+           "currentVersionedKeyIdentifier": "https://myvault8569.vault.azure.net/keys/wrappingKey/0682afdd9c104f4285df20107e956cad",
+           "lastKeyRotationTimestamp": "2019-12-13T20:36:23.7023290Z"
+          },
+        "services": {
+           "file": {
+             "keyType": "Account",
+             "enabled": true,
+             "lastEnabledTime": "2020-12-15T00:43:14.1739587Z"
+           },
+          "blob": {
+             "keyType": "Account",
+             "enabled": true,
+             "lastEnabledTime": "2020-12-15T00:43:14.1739587Z"
+           }
+         },
+         "keySource": "Microsoft.Keyvault"
+        },
+        "accessTier": "Hot",
+        "provisioningState": "Succeeded",
+        "creationTime": "2020-12-15T00:43:14.0839093Z",
+        "primaryEndpoints": {
+          "web": "https://sto4445.web.core.windows.net/",
+          "dfs": "https://sto4445.dfs.core.windows.net/",
+          "blob": "https://sto4445.blob.core.windows.net/",
+          "file": "https://sto4445.file.core.windows.net/",
+          "queue": "https://sto4445.queue.core.windows.net/",
+          "table": "https://sto4445.table.core.windows.net/"
+        },
+        "primaryLocation": "eastus",
+        "statusOfPrimary": "available"
+       }
+      }
+   },
+    "202": {}
```

Comment:
Update account will return 200, see [link](https://github.com/Azure/azure-rest-api-specs/blob/0c90247ad2604b12457a43e7613bc07c709795ad/specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json#L296), so we can't use this sample for update account API.

Reply:
And please only input the required parameter  for update user identity in update account sample.

Reply:
Thanks WeiWei. I have made that change in the update example.